extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -65,12 +66,29 @@ export class AmpAdUIHandler {
    * Order: try collapse -> apply provided fallback -> apply default fallback
    */
   applyNoContentUI() {
-    if (getAdContainer(this.element_) == 'AMP-STICKY-AD') {
+    if (getAdContainer(this.element_) === 'AMP-STICKY-AD') {
       // Special case: force collapse sticky-ad if no content.
       this.baseInstance_./*OK*/collapse();
       return;
     }
 
+    if (getAdContainer(this.element_) === 'AMP-FX-FLYING-CARPET') {
+
+      // Special case: force collapse ad if only child of a flying carpet.
```

Comment:
nit: if it is the only child : )

And could we add comment on the discussion we had about `<amp-layout>`.

Reply:
Oops! Yeah I forgot thank you!